spec/PeriodicBackgroundSync-index.bs

```diff
@@ -93,3 +93,11 @@ The <dfn>tag</dfn> is a DOMString. Within one [=list of periodicsync registratio
 The options are a dictionary containing [=minInterval=], a long long. <dfn>minInterval</dfn> is used to specify the minimum interval, in milliseconds, at which the periodic synchronization should happen.
 
 The <dfn>registration state</dfn> is one of <dfn>pending</dfn>, <dfn>waiting</dfn>, <dfn>firing</dfn>, or <dfn>reregisteredWhileFiring</dfn>. It is initially set to [=pending=].
+
+
+Resource Usage {#resources}
+============================
+A website will most likely download resources from the network when processing a <code>periodicsync</code> event. The underlying operating system MAY launch the user agent to dispatch these events, and MAY keep it awake for a pre-defined duration to allow processing of the events. Both cause battery drain.
+The user agent SHOULD cap the frequency of these events to limit resource usage by websites when the user has navigated away.
+
+In addition, the user agent SHOULD consider other factors such as user engagement with the origin, and any user indications to temporarily reduce data consumption, to adjust the frequency of <code>periodicsync</code> events.
```

Comment:
It's a bit hard to have a _SHOULD_ requirement without any of the subsequent suggestions being specified. I would keep this non-normative and mark it as such, which also allows you to be a bit more expressive. (E.g. specifically mention Data Saving Mode).

Reply:
Done. In your knowledge, does any spec export a definition of Data Saving Mode I could link to? I think I should avoid linking to this: https://source.android.com/devices/tech/connect/data-saver

Reply:
We shouldn't link to that as it's Android specific.

Closest would be the Network Information API AFAIK, but they don't define where the value comes from, or what it means: https://wicg.github.io/netinfo/#savedata-attribute

Reply:
Thanks! Yes that is indeed pointing to something the browser sets but not defining what the source of that info is. I think I'll skip linking to anything.

Reply:
I find it a bit odd that a non-normative section has normative requirements (MUST/SHOULD). I'm not sure what the best practices are here surrounding that. Other than that, LGTM

Reply:
According to https://tabatkins.github.io/bikeshed/#metadata-informative-classes, I should see errors/warnings when I try to parse this section if I have indeed included definitions or requirements here. I don't see any.

There is no "MUST" here, and I think SHOULD is fine in these sections. Let me know if you disagree.

Reply:
Why do you think SHOULD is fine in these sections?

Reply:
Because bikeshed doesn't "Complain About: accidental-2119 metadata" when parsing the spec, unlike what the documentation suggests:
https://tabatkins.github.io/bikeshed/#metadata-informative-classes

However, looking at RFC2119: https://tools.ietf.org/html/rfc2119, SHOULD is indeed a level of requirement.

Marking this "non-normative" was a suggestion by you, so we can mention "Data Saver" mode specifically. If we don't want to mention SHOULD here, we have two paths forward:
1. I can split this out in two sections, one non-normative, one normative.
2. We keep this normative, and use the non-specific terminology from the explainer: https://github.com/WICG/BackgroundSync/blob/master/explainers/periodicsync-explainer.md#power-and-network-conditions

I'm leaning towards 2, but let me know what you think.

Reply:
You could just use "should" in lower case, that is fine. It's the upper case variants that impose requirements.

Reply:
done.